Add link to GL group variables KB on Sample configs (#1425)
semgrep/semgrep-docs: docs/semgrep-ci/sample-ci-configs.md

<GlcicdSemgrepAppSast />

You can **run specific product scans** by passing an argument, such as `--supply-chain`. View the [list of arguments](/getting-started/cli/#scan-using-specific-semgrep-products).

> Prefer to use GitLab group variables? See [this guide](/docs/kb/semgrep-code/gitlab-group-variables/) for an appropriate configuration.

Reviewer: ```suggestion
Prefer to use GitLab group variables? See a [sample configuration](/docs/kb/semgrep-code/gitlab-group-variables/).
```
